web3.js/test/connection.test.ts

```diff
@@ -1513,6 +1513,128 @@ describe('Connection', function () {
     expect(result).to.be.empty;
   });
 
+  it('get block height', async () => {
+    const commitment: Commitment = 'confirmed';
+
+    await mockRpcResponse({
+      method: 'getBlockHeight',
+      params: [{commitment: commitment}],
+      value: 10,
+    });
+
+    const blockHeight = await connection.getBlockHeight(commitment);
+    expect(blockHeight).to.be.a('number');
+  });
+
+  it('get block production', async () => {
+    const commitment: Commitment = 'processed';
+
+    // Find slot of the lowest confirmed block
+    await mockRpcResponse({
+      method: 'getFirstAvailableBlock',
+      params: [],
+      value: 1,
+    });
+    let firstSlot = await connection.getFirstAvailableBlock();
+
+    // Find current block height
+    await mockRpcResponse({
+      method: 'getBlockHeight',
+      params: [{commitment: commitment}],
+      value: 10,
+    });
+    let lastSlot = await connection.getBlockHeight(commitment);
+
+    const blockProductionConfig = {
+      commitment: commitment,
+      range: {
+        firstSlot: firstSlot,
+        lastSlot: lastSlot,
```

Comment:
Tip: when the key and value are the same, you can just write:

```suggestion
        firstSlot,
        lastSlot,
```